### PR TITLE
Extended title for ADR 0013

### PR DESCRIPTION
**Pull request by trmartin4**
## Objective

The title for ADR 0013 didn't include "for request/response models". It wasn't clear from the title what the scope of the change was, especially when compared with 0011 ("Angular folder structure").

If there is a hesitancy to extend the ADR title to a third line (shown below), perhaps we could include phrase it as "Request/response model folder structure", as the "request/response model" is important context to have.

image

### CODE REVIEW

**Review by MGibson1 — COMMENTED** [review]
Seems reasonable to me, @Hinton what do you think? IIRC this title is off because you wanted a bigger scope but we limited ourselves

**Comment by trmartin4** [comment]
Perhaps to maintain consistency and clearly delineate the scope, we could title it "Service request/response folder structure" to parallel "Angular folder structure"

**Comment by Hinton** [comment]
Perhaps we could rename it to `Deprecate global request/response models`?

**Comment by trmartin4** [comment]
@Hinton that sounds great!  I've updated it.

**Review by Hinton — COMMENTED** [review]
We should change the filename while we're at it

**Comment by trmartin4** [comment]
> We should change the filename while we're at it

Done.  I adjusted the title in the document as well.

**Review by Hinton — APPROVED** [review]

File: docs/architecture/adr/0013-deprecate-global-request-response-models.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 0013 - Avoid layered folder structure
+title: 0013 - Deprecate global request/response models
```
Comment (Hinton) [inline docs/architecture/adr/0013-deprecate-global-request-response-models.md:2]: We can remove the title since it's identical to the header below.

**Review by Hinton — CHANGES_REQUESTED** [review]

File: docs/architecture/adr/0013-deprecate-global-request-response-models.md (hunk 1 above)
Comment (Hinton) [inline docs/architecture/adr/0013-deprecate-global-request-response-models.md:2]:
```suggestion
```

**Review by Hinton — APPROVED** [review]

**Pull request merged**